website/client/src/assets/scss/markdown.scss

```
@@ -41,4 +41,9 @@
   img {
     max-width: 100%;
   }
+  
+  blockquote {
+    padding: 0 16px;
```

```suggestion
    padding-left: 16px;
```

Or was the additional `padding-right` also by design?

---

I tried playing with using different padding for left and right for a little bit just before I settled on this, but I felt like it looked "wrong" on my local install in regards to where the words wrapped, when compared to what I was looking at on Zeplin. (I tried out 16 padding on the left, alone, as well as 16 left and a smaller one on the right, based on what I thought the numbers on Zeplin were suggesting.)

Basically, I settled on what I used because it appeared to be most faithful to the example in terms of appearance. ._.

(Sidenote: It's also incredibly similar to the pre-2017-redesign padding (which was the same padding code but 15px), though that doesn't really play into why I went with it...)

---

Cool. Aren't you going to run in to the issue that you're basically running out of space for words if you nest enough blockquotes? Or is there a limit to the amount of nesting for blockquotes?

---

We can probably rely on common sense to avoid that most of the time. I'd suggest ignoring the problem for the sake of simpler code (unless it becomes obvious that it happens often enough for us to care).

---

Sounds good.

---

I know the conversation has more or less reached a resolution (impasse? undetstanding? IDK?) but: Staring at my screenshot right now, I think removing right padding would only allow one extra level of quote if extreme nesting over the current code anyway, based on where the words wrap. OTOH, I might also be tired with little time (gotta get some house cleaning done) to fire up my installation and *confirm* that and thereby am working entirely off the way my tired mind is parsing the visual alone.

---

Sorry I'm a little late to the conversation, the original issue mentioned that we wouldn't be allowing nesting for the first pass. Thanks for Citrus' design eye, I thought nesting looked good and don't mind supporting it. This looks good. 👍

---

To be fair regarding my "design eye", I haven't included any nesting-*specific* code (the markdown already supported nesting the element), I just remembered halfway through testing it out that nesting is a thing that can usually be done so I just tested it to be sure it a) worked the way I thought it did, and b) wasn't a big ugly mess, heh.

In all honesty, every time I code anything I'm really just pleasantly surprised that it doesn't explode?